scripts/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -20,6 +20,11 @@ type BehaviorMenuItem = {
   };
 };
 
+async function importFile(pathDir, name) {
+  const path = `${pathDir}\\${name}.ts`;
```

Comment:
You should probably use `path.join` here since backslashes are Windows-specific

Reply:
thanks, implemented proposal change